src/blocks/collapsible/collapsible.js

```diff
@@ -0,0 +1,203 @@
+/**
+ * BLOCK: collapsible
+ *
+ * A collapsible content block.
+ */
+
+// External dependencies.
+import classnames from 'classnames';
+
+// Import CSS.
+import './style.scss';
+import './editor.scss';
+
+// WordPress dependencies.
+const { __ } = wp.i18n;
+const { registerBlockType, createBlock } = wp.blocks;
+const {
+	PanelBody,
+	ToggleControl,
+	SelectControl,
+	TextControl,
+	RangeControl,
+	Path,
+	SVG,
+	Button,
+	ButtonGroup,
+} = wp.components;
+const {
+	InnerBlocks,
+	BlockControls,
+	InspectorControls,
+	RichText
+} = wp.blockEditor;
+const { select } = wp.data;
+const { getBlocks } = select( 'core/block-editor' );
+
+import HeadingToolbar from '../headline/heading-toolbar';
+
+// Register the block.
+registerBlockType( 'editorial/collapsible', {
+	name: 'editorial/collapsible',
```

Comment:
This will probably mean doing a find/replace across all the files to change it from `editorial/collapsible` to `bu/collapsible` and changing the class names in all the files.

Reply:
Updated. Thank you